tests/test_train_classifer.py

```diff
@@ -0,0 +1,161 @@
+import os
+from os import path
+from opentutor_classifier import AnswerClassifierInput
+from opentutor_classifier.svm import (
+    train_classifier,
+    train_default_classifier,
+    SVMAnswerClassifier,
+    load_config_into_objects,
+)
+import pytest
+from typing import Dict, Tuple
+from . import fixture_path
+
+
+@pytest.fixture(scope="module")
+def data_root() -> str:
+    return fixture_path("data")
+
+
+@pytest.fixture(scope="module")
+def shared_root() -> str:
+    return fixture_path("shared")
+
+
+def __train_model(
+    tmpdir, data_root: str, shared_root: str
+) -> Tuple[str, Dict[int, int]]:
+    output_dir = os.path.join(
+        tmpdir.mkdir("test"),
+        "model_root",
+        os.path.basename(os.path.normpath(data_root)),
+    )
+    accuracy = train_classifier(
+        data_root=data_root, shared_root=shared_root, output_dir=output_dir
+    )
+    return output_dir, accuracy
+
+
+def test_outputs_models_at_specified_model_root_for_q1(
+    tmpdir, data_root: str, shared_root: str
+):
+    output_dir, _ = __train_model(
+        tmpdir, path.join(data_root, "question1"), shared_root
+    )
+    assert path.exists(path.join(output_dir, "models_by_expectation_num.pkl"))
+    assert path.exists(path.join(output_dir, "config.yaml"))
+
+
+def __train_default_model(
+    tmpdir, data_root: str, shared_root: str
+) -> Tuple[str, Dict[int, int]]:
+    output_dir = os.path.join(
+        tmpdir.mkdir("test"),
+        "model_root",
+        os.path.basename(os.path.normpath(data_root)),
+    )
+    accuracy = train_default_classifier(
+        data_root=data_root, shared_root=shared_root, output_dir=output_dir
+    )
+    return output_dir, accuracy
+
+
+def test_outputs_models_at_specified_model_root_for_default_model(
+    tmpdir, data_root: str, shared_root: str
+):
+    output_dir, _ = __train_default_model(tmpdir, data_root, shared_root)
+    assert path.exists(path.join(output_dir, "models_by_expectation_num.pkl"))
+
+
+def test_outputs_models_at_specified_model_root_for_q2(
```

Comment:
similar comment to previous test file: refactor tests that are `for_q[1|2]` into a single paramterized test function.

Probably is ok/best to leave the tests about `default` model as separate, since the code really is different there

Reply:
I have also refactored `test_cli_outputs_models_at_specified_model_root_for_[q1|q2]` into a single paramterized test function :)